service/mapper/state.go

```diff
@@ -32,11 +32,12 @@ type State struct {
 	done      chan struct{}
 }
 
-// EmptyState returns a new empty state that uses the given Forest.
+// EmptyState returns a new empty state that uses the given forest.
```

Comment:
I tend to capitalize words that are interfaces/types because my IDE understands it and makes those words control-clickable/highlighted which is neat. I'm fine with not doing it though, but just to say that this was not a typo but intentional :p

Reply:
That's fair enough, but I don't think it necessarily makes sense when you have the parameters right below. Personally, I try to keep the comments in good English, and when I want to explicitly refer to a type, I will use back ticks, like so: `Forest`.